src/read_utils.jl

```diff
@@ -57,6 +57,21 @@ function analyze_symmetry_data(
     return band_summaries
 end
 
+function fix_gamma_irrep(symeigsd::Dict{String, Vector{Vector{ComplexF64}}}, lgd::Dict{String, LittleGroup{D}}, mode::AbstractString) where D
+    symeigsd_fixed = symeigsd
+    lg_gamma = lgd["Γ"]
+    signs = [det(rotation(g)) for g in lg_gamma]
+    if (D == 2)
+        if (mode == "tm")
+            symeigsd_fixed["Γ"][1] = ones(length(signs))
+        else (mode == "te")
+            symeigsd_fixed["Γ"][1] = signs
+        end
+    elseif (D ==3)
+        symeigsd_fixed[Γ][1] = signs .* (2cos.(2π ./ Crystalline.rotation_order.(lg_gamma)) .+ 1) .- 1
```

Comment:
This needs to be split over the first two bands, not just the first one. How we choose to split it seems a little iffy, but it probably cannot matter (since the two bands must always touch anyway - so the splitting is ill-definite regardless; i.e., cannot matter how we choose to split).
Maybe we just split 35% in one and 65% in another: the reason we shouldn't do an even 50%/50% split is it could make the bands look separable (which we know they are not) - we'd want to force them together, and an "ugly" break could do that.
So, maybe we try something like this instead:
```jl
x2T = signs .* (2cospi.(2 ./ Crystalline.rotation_order.(lg)) .+ 1) .- 1
ugliness = 0.3123
symeigsd_fixed[Γ][1] = ugliness * x2T
symeigsd_fixed[Γ][2] = (1- ugliness) * x2T
```